diacritics_test.go

```diff
@@ -61,7 +61,7 @@ func runSearch(t *testing.T, ignorecase, smartcase, ignorediacritics, smartdiacr
 	gOpts.smartcase = smartcase
 	gOpts.ignoredia = ignorediacritics
 	gOpts.smartdia = smartdiacritics
-	matched, _ := searchMatch(base, pattern)
+	matched, _ := searchMatch(base, pattern, gOpts.globsearch)
```

Comment:
I would just hardcode this to `false` since this test doesn't deal with globs anyway:

```suggestion
	matched, _ := searchMatch(base, pattern, false)
```